apps/meteor/client/components/Omnichannel/CustomFieldsForm/CustomTextInput.tsx

```diff
@@ -0,0 +1,49 @@
+import { Field, TextInput } from '@rocket.chat/fuselage';
+import type { OmnichannelCustomFieldEndpointPayload } from '@rocket.chat/rest-typings';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import React, { forwardRef, ReactElement, useMemo } from 'react';
+import { useFormContext, UseFormRegisterReturn } from 'react-hook-form';
+
+export const CustomTextInput = forwardRef<
+	HTMLInputElement,
+	UseFormRegisterReturn & {
+		data: OmnichannelCustomFieldEndpointPayload;
+	}
+>(function CustomTextInput({ name, data, onChange }, ref): ReactElement {
+	const t = useTranslation();
+	const {
+		formState: { errors },
+	} = useFormContext();
+
+	const errorMessage = useMemo(() => {
+		if (errors?.livechatData && errors?.livechatData[data._id]?.type === 'regexp') {
```

Comment:
I think here you should assign `errors?.livechatData && errors?.livechatData[data._id]` to a variable

Reply:
ok